Sources/StreamChat/Database/DTOs/ChannelListQueryDTO.swift

```diff
@@ -38,12 +40,12 @@ extension NSManagedObjectContext {
     }
     
     func saveQuery(query: ChannelListQuery) -> ChannelListQueryDTO {
-        if let existingDTO = ChannelListQueryDTO.load(filterHash: query.filter.filterHash, context: self) {
+        let request = ChannelListQueryDTO.fetchRequest(keyPath: "filterHash", equalTo: query.filter.filterHash)
+        if let existingDTO = ChannelListQueryDTO.load(by: request, context: self).first {
```

Comment:
Shouldn't it use the method changed above? `static func load(filterHash: String, context: NSManagedObjectContext) -> ChannelListQueryDTO?`

Reply:
same reason as in `AttachmentDTO`. this case, lines 43-49 act like `loadOrCreate`, so calling `load(filterHash)` would cause another `request` to be created, which is not necessary since we have the `request` already created for `insertNewEntity`